### ISSUE TITLE
Refresh Santa section

### ISSUE THREAD

**Issue by drduh** [drduh/macOS-Security-and-Privacy-Guide#455]
It is more than a few years old at this point and likely needs a complete overhaul.

**Comment by beerisgood** [comment]
From your guide:

> Santa can be used to only allow trusted code execution, or to blacklist known malware from executing on a Mac

This is already an macOS built-in feature called [Gatekeeper](https://support.apple.com/guide/security/sec5599b66df/).
There is no need to increase the attack surface with a external program.

**Comment by kimg45** [comment]
Gatekeeper only covers apps while Santa covers all binaries. It does provide functionality that isn't there in the base operating system, although I think it's most useful in a managed environment like at a company or something.

**Comment by friadev** [comment]
Looks like Google Santa is no longer maintained unfortunately.

**Comment by drduh** [comment]
too bad - has anyone given the fork(s) a chance?

**Comment by headmin** [comment]
@drduh yes, the whole team left Google and founded a new company - [North Pole Security](https://northpole.security/).
 
The new updated version of Santa (last 2025.3 release beginning of this week)  has gained more features and is in active development, with more to come. 

https://github.com/northpolesec/santa

**Comment by drduh** [comment]
that's great news - let's keep this issue open in hope of future contributions detailing its use.
